app/bundles/CoreBundle/Helper/AbstractFormFieldHelper.php

```diff
@@ -152,7 +152,7 @@ public static function parseList($list, $removeEmpty = true, $ignoreNumerical =
                     $label = $val;
                 }
 
-                $choices[trim(html_entity_decode($val, ENT_QUOTES))] = trim(html_entity_decode($val, ENT_QUOTES));
+                $choices[trim(html_entity_decode($label, ENT_QUOTES))] = trim(html_entity_decode($val, ENT_QUOTES));
```

Comment:
I think maybe variables need to be reversed shouldnt it be 
`$choices[$val] => $label`
Else it will just inverse the values every save?
Or is there coding in the mobile area there that requires that order?
Since in custom fields this code inverses the fields every save.